src/com/jetbrains/Deadline.java

```diff
@@ -1,18 +1,29 @@
 package com.jetbrains;
 
+import java.lang.Throwable;
 public class Deadline extends Task {
-    String dueDate;
```

Comment:
I think dueDate should be used instead of by as the former is more understandable!